src/encoded/types/dataset.py

```diff
@@ -466,6 +466,48 @@ def files(self, request, original_files, status):
     def superseded_by(self, request, superseded_by):
         return paths_filtered_by_status(request, superseded_by)
 
+    @calculated_property(condition='contributing_files', schema={
+        "title": "Biochemical profile inputs",
+        "description": "The data used to generate a cCRE annotation for a given cell type or biosample.",
+        "type": "string",
+        "notSubmittable": True,
+
+    })
+    def biochemical_inputs(
+        self,
+        request,
+        annotation_type,
+        encyclopedia_version=None,
+        contributing_files=None
+    ):
+        # https://encodedcc.atlassian.net/browse/ENCD-5288
+        inputs_list = []
+        if encyclopedia_version is not None and \
+            encyclopedia_version in (
+                'ENCODE v4', 'ENCODE v5', 'ENCODE v6'
+                ):
+            if annotation_type == 'candidate Cis-Regulatory Elements':
+                if contributing_files is not None:
+                    for input_file in contributing_files:
+                        file = request.embed(input_file, '@@object')
+                        if file['dataset']:
```

Comment:
I think here you can skip calculated properties?
request.embed(input_file, '@@object?skip_calculated=true'

Reply:
You are right, I added your suggestion (and it still appears fine on local app)